roles/ceph-rgw/defaults/main.yml

```diff
@@ -41,6 +36,31 @@ rgw_pull_proto: "http"
 #    pg_num: 32
 #  foo:
 #    pg_num: 4
+#
+
+
+#############
+# MULTISITE #
+#############
+
+rgw_multisite: false
+
+# The following Multi-site related variables should be set by the user.
+# rgw_zone is set to default to enable compression for clusters configured without rgw multi-site
+rgw_zone: default
+
+rgw_zonemaster: true
+rgw_zonesecondary: false
+#rgw_zonegroup: dummy # should be set by the user
```

Comment:
you can leave that declared with a default `dummy`, that's fine I guess. If this is a must-have/something that we need to check before deploying then you should add it in the ceph-validate role.

Reply:
If `rgw_multisite` is `true` then all of the variables in the multisite section need to be declared. I'm not sure if `dummy` means something to ansible, but I think the reason it is there is because I it was used as a filler in other parts of Ceph ansible. 

Given that, is there something I should change? Is there an example in the ceph-validate role you can point me to?

Reply:
I just think it's a common practice to have all the variables declared so you can use them in your playbook without having to make sure they are defined or not. Typically you don't check their existence (if someone forgets to enable it) but its value only.